labtool2.0/src/tests/__snapshots__/BrowseReviews.test.js.snap

```diff
@@ -67,7 +67,7 @@ exports[`<BrowseReviews /> BrowseReviews Component should render correctly 1`] =
         <p
           className="noPrevious"
         >
-          First time to participate the course
+          Has not been on this course before
```

Comment:
"Has not taken part in/on this course before"?

Reply:
Seems better